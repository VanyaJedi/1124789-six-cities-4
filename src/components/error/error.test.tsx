import * as React from "react";
import * as renderer from "react-test-renderer";
import Error from "./error";

it(`Error`, () => {
  const tree = renderer
    .create(
        <Error/>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
