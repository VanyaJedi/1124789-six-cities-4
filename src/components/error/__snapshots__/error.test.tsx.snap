// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error 1`] = `
<div
  className="error"
  style={
    Object {
      "border": "2px solid red",
      "height": "100px",
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": "200px",
    }
  }
>
  <span>
    Сервер недоступен
  </span>
</div>
`;
